apps/web/src/components/GalleryEditor/GalleryOnboardingGuide/OnboardingDialogContext.tsx

```diff
@@ -0,0 +1,117 @@
+import { createContext, useCallback, useContext, useMemo, useState } from 'react';
+import { graphql, useFragment } from 'react-relay';
+
+import { OnboardingDialogContextFragment$key } from '~/generated/OnboardingDialogContextFragment.graphql';
+
+import useUpdateUserExperience from './useUpdateUserExperience';
+
+type OnboardingDialogContextType = {
+  step: number;
+  dialogMessage: string;
+  nextStep: () => void;
+  handleClose: () => void;
+};
+
+const OnboardingDialogMessage = {
+  1: 'You can modify your gallery name and description here.',
+  2: 'You can display multiple collections in a gallery. Name your first collection title here, and add a description.',
+  3: 'Search or filter through various chains and wallets to find the pieces you want to curate',
+  4: 'Click a piece to add to your collection.',
+  5: 'You can add multiple sections to adjust the number of columns and display your pieces in creative ways',
```

Comment:
let's add a period `.` to the end of step 3 and step 5

Reply:
for step 5, let's reword "display" -> "showcase"

Reply:
Updated!